components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -465,16 +464,9 @@ public DataSection genesBody(Patient patient) throws Exception
         }
 
         List<String> fields = Arrays.asList("status", "strategy", "comments");
-        Map<String, String> valueTranslates = new HashMap<String, String>();
-        valueTranslates.put("sequencing", "Sequencing");
-        valueTranslates.put("deletion", "Deletion/duplication");
-        valueTranslates.put("familial_mutation", "Familial mutation");
-        valueTranslates.put("common_mutations", "Common mutations");
-        valueTranslates.put("solved", "Confirmed causal");
-        valueTranslates.put("rejected", "Negative");
-        valueTranslates.put("candidate", "Candidate");
-
-        DataCell cell = new DataCell("Genes", 0, y);
+        List<String> strategyTranslates =
+            Arrays.asList("sequencing", "deletion", "familial_mutation", "common_mutations");
```

Comment:
What about "solved"/"rejected"/"candidate"? This is a big commit and I may have missed those, or maybe the gene/variant design has changed?